cpp/include/cudf/utilities/type_dispatcher.hpp

```diff
@@ -137,6 +137,12 @@ MAP_NUMERIC_SCALAR(float)
 MAP_NUMERIC_SCALAR(double)
 MAP_NUMERIC_SCALAR(cudf::experimental::bool8)
 
+template <>
+struct type_to_scalar_type_impl<std::string> {
```

Comment:
I couldn't find this being used in this PR.

Reply:
It's used by the conversion operator you called out in this comment: https://github.com/rapidsai/cudf/pull/3657#discussion_r360969639